{
 "about": {
  "channels": [
   "https://conda.anaconda.org/conda-forge/osx-64",
   "https://conda.anaconda.org/conda-forge/noarch",
   "https://repo.continuum.io/pkgs/free/osx-64",
   "https://repo.continuum.io/pkgs/free/noarch",
   "https://repo.continuum.io/pkgs/pro/osx-64",
   "https://repo.continuum.io/pkgs/pro/noarch"
  ],
  "conda_build_version": "2.1.10",
  "conda_env_version": "4.2.13",
  "conda_private": false,
  "conda_version": "4.2.13",
  "env_vars": {
   "CIO_TEST": "<not set>",
   "CONDA_DEFAULT_ENV": "root",
   "CONDA_ENVS_PATH": "<not set>",
   "DYLD_LIBRARY_PATH": "<not set>",
   "PATH": "/Users/travis/miniconda3/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p643/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p643@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p643/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin",
   "PYTHONHOME": "<not set>",
   "PYTHONPATH": "<not set>"
  },
  "home": "http://docs.pyinvoke.org",
  "license": "BSD-2-Clause",
  "offline": false,
  "root_pkgs": [
   "conda-forge::anaconda-client-1.6.3-py36_0",
   "asn1crypto-0.22.0-py36_0",
   "conda-forge::beautifulsoup4-4.5.3-py36_0",
   "cffi-1.10.0-py36_0",
   "conda-forge::chardet-3.0.2-py36_1",
   "conda-forge::clyent-1.2.1-py36_0",
   "conda-forge::conda-4.2.13-py36_0",
   "conda-forge::conda-build-2.1.10-py36_0",
   "conda-forge::conda-env-2.6.0-0",
   "conda-forge::conda-forge-build-setup-4.4.4-0",
   "conda-forge::conda-verify-2.0.0-py36_0",
   "cryptography-1.8.1-py36_0",
   "conda-forge::filelock-2.0.6-py36_0",
   "idna-2.5-py36_0",
   "conda-forge::jinja2-2.9.5-py36_0",
   "conda-forge::markupsafe-1.0-py36_0",
   "openssl-1.0.2l-0",
   "packaging-16.8-py36_0",
   "pip-9.0.1-py36_1",
   "conda-forge::pkginfo-1.2.1-py36_0",
   "pycosat-0.6.2-py36_0",
   "pycparser-2.17-py36_0",
   "conda-forge::pycrypto-2.6.1-py36_1",
   "pyopenssl-17.0.0-py36_0",
   "pyparsing-2.1.4-py36_0",
   "python-3.6.1-2",
   "conda-forge::python-dateutil-2.6.0-py36_0",
   "conda-forge::pytz-2017.2-py36_0",
   "conda-forge::pyyaml-3.12-py36_1",
   "readline-6.2-2",
   "requests-2.14.2-py36_0",
   "ruamel_yaml-0.11.14-py36_1",
   "setuptools-27.2.0-py36_0",
   "six-1.10.0-py36_0",
   "sqlite-3.13.0-0",
   "tk-8.5.18-0",
   "wheel-0.29.0-py36_0",
   "xz-5.2.2-1",
   "yaml-0.1.6-0",
   "zlib-1.2.8-3"
  ],
  "summary": "Pythonic task execution"
 },
 "conda_build_config": {},
 "files": [
  "lib/python3.5/site-packages/invoke/vendor/yaml3/events.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/constructor.py",
  "bin/invoke",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/composer.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/cyaml.py",
  "lib/python3.5/site-packages/invoke/parser/context.py",
  "lib/python3.5/site-packages/invoke/vendor/fluidity/backwardscompat.py",
  "lib/python3.5/site-packages/invoke/parser/parser.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/resolver.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/parser.py",
  "lib/python3.5/site-packages/invoke/__init__.py",
  "lib/python3.5/site-packages/invoke/vendor/fluidity/__init__.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/tokens.py",
  "lib/python3.5/site-packages/invoke/parser/argument.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/loader.py",
  "lib/python3.5/site-packages/invoke/vendor/__init__.py",
  "lib/python3.5/site-packages/invoke/tasks.py",
  "lib/python3.5/site-packages/invoke/config.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/error.py",
  "lib/python3.5/site-packages/invoke/runners.py",
  "lib/python3.5/site-packages/invoke/context.py",
  "lib/python3.5/site-packages/invoke/program.py",
  "lib/python3.5/site-packages/invoke/env.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/__init__.py",
  "lib/python3.5/site-packages/invoke/vendor/decorator.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/reader.py",
  "lib/python3.5/site-packages/invoke/util.py",
  "lib/python3.5/site-packages/invoke/vendor/lexicon/__init__.py",
  "lib/python3.5/site-packages/invoke/platform.py",
  "lib/python3.5/site-packages/invoke/vendor/lexicon/attribute_dict.py",
  "lib/python3.5/site-packages/invoke/collection.py",
  "lib/python3.5/site-packages/invoke/vendor/lexicon/alias_dict.py",
  "lib/python3.5/site-packages/invoke/watchers.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/emitter.py",
  "lib/python3.5/site-packages/invoke/exceptions.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/scanner.py",
  "lib/python3.5/site-packages/invoke/vendor/six.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/dumper.py",
  "lib/python3.5/site-packages/invoke/vendor/fluidity/machine.py",
  "lib/python3.5/site-packages/invoke/main.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/representer.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/serializer.py",
  "lib/python3.5/site-packages/invoke/parser/__init__.py",
  "lib/python3.5/site-packages/invoke/_version.py",
  "lib/python3.5/site-packages/invoke-0.18.1-py3.5.egg-info/PKG-INFO",
  "lib/python3.5/site-packages/invoke/executor.py",
  "lib/python3.5/site-packages/invoke/complete.py",
  "lib/python3.5/site-packages/invoke/vendor/yaml3/nodes.py",
  "lib/python3.5/site-packages/invoke/loader.py",
  "bin/inv",
  "lib/python3.5/site-packages/invoke/__main__.py"
 ],
 "index": {
  "arch": "x86_64",
  "build": "py35_0",
  "build_number": 0,
  "depends": [
   "python 3.5*"
  ],
  "license": "BSD-2-Clause",
  "name": "invoke",
  "platform": "osx",
  "subdir": "osx-64",
  "version": "0.18.1"
 },
 "metadata_version": 1,
 "name": "invoke",
 "raw_recipe": "{% set version = \"0.18.1\" %}\n\npackage:\n  name: invoke\n  version: {{ version }}\n\nsource:\n  fn: invoke-{{ version }}.tar.gz\n  url: https://pypi.io/packages/source/i/invoke/invoke-{{ version }}.tar.gz\n  sha256: a239148c4cd9ff91a103204472599148007600424520ec1dcf2e8c7826531902\n\nbuild:\n  number: 0\n  script: python setup.py install --single-version-externally-managed --record=record.txt\n  entry_points:\n    - invoke = invoke.main:program.run\n    - inv = invoke.main:program.run\n\nrequirements:\n  build:\n    - python\n    - setuptools\n\n  run:\n    - python\n\ntest:\n  # Python imports\n  imports:\n    - invoke\n    - invoke.parser\n    - invoke.vendor\n    - invoke.vendor.fluidity\n    - invoke.vendor.lexicon\n    - invoke.vendor.yaml2    # [py2k]\n    - invoke.vendor.yaml3    # [py3k]\n\n  commands:\n    - invoke --help\n    - inv --help\n\nabout:\n  home: http://docs.pyinvoke.org\n  license: BSD-2-Clause\n  summary: 'Pythonic task execution'\n\nextra:\n  recipe-maintainers:\n    - ivoflipse\n    - Korijn\n    - nicoddemus\n",
 "rendered_recipe": {
  "about": {
   "home": "http://docs.pyinvoke.org",
   "license": "BSD-2-Clause",
   "summary": "Pythonic task execution"
  },
  "build": {
   "entry_points": [
    "invoke = invoke.main:program.run",
    "inv = invoke.main:program.run"
   ],
   "noarch": "",
   "noarch_python": false,
   "number": "0",
   "script": "python setup.py install --single-version-externally-managed --record=record.txt",
   "string": "py35_0"
  },
  "extra": {
   "final": true,
   "recipe-maintainers": [
    "ivoflipse",
    "Korijn",
    "nicoddemus"
   ]
  },
  "package": {
   "name": "invoke",
   "version": "0.18.1"
  },
  "requirements": {
   "build": [
    "ca-certificates 2017.4.17 0",
    "certifi 2017.4.17 py35_0",
    "ncurses 5.9 10",
    "openssl 1.0.2k 0",
    "python 3.5.3 3",
    "readline 6.2 0",
    "setuptools 33.1.1 py35_0",
    "sqlite 3.13.0 1",
    "tk 8.5.19 1",
    "xz 5.2.2 0",
    "zlib 1.2.11 0"
   ],
   "run": [
    "python"
   ]
  },
  "source": {
   "fn": "invoke-0.18.1.tar.gz",
   "sha256": "a239148c4cd9ff91a103204472599148007600424520ec1dcf2e8c7826531902",
   "url": "https://pypi.io/packages/source/i/invoke/invoke-0.18.1.tar.gz"
  },
  "test": {
   "commands": [
    "invoke --help",
    "inv --help"
   ],
   "imports": [
    "invoke",
    "invoke.parser",
    "invoke.vendor",
    "invoke.vendor.fluidity",
    "invoke.vendor.lexicon",
    "invoke.vendor.yaml3"
   ]
  }
 },
 "version": "0.18.1"
}